geocoder_tester/world/france/test_from_user_input.py

```diff
@@ -0,0 +1,99 @@
+from geocoder_tester.base import assert_search
+import pytest
+
+
+@pytest.mark.qwant_non_regression
+def test_ajaccio():
+    assert_search("ajaccio", {'name': 'Ajaccio', 'coordinate': '41.9263991, 8.7376029,5000'})
+
+@pytest.mark.qwant_non_regression
+def test_chateauroux():
+    assert_search("chateauroux", {'name': 'Châteauroux', 'postcode': '36000'})
+
+@pytest.mark.qwant_non_regression
+def test_indre_et_loire():
+    assert_search("indre et loire", {'name': 'Indre-et-Loire', 'coordinate': '47.2232046,0.6866702523286876,50000'})
+
+@pytest.mark.qwant_non_regression
+def test_28_Rue_de_la_Belle_Marie_Barbizon():
+    assert_search("28 Rue de la Belle Marie Barbizon", {'street': 'Rue de la Belle Marie', 'postcode': '77630'})
+
+@pytest.mark.qwant_non_regression
+def test_6_Rue_Romy_Schneider_Chevry_Cossigny():
+    assert_search("6 Rue Romy Schneider Chevry-Cossigny", {'name': '6 Rue Romy Schneider', 'postcode': '77173'})
+
+@pytest.mark.qwant_non_regression
+def test_25_Promenade_des_Golfeurs_77600():
+    assert_search("25 Promenade des Golfeurs 77600", {'housenumber': '25', 'street': 'Promenade des Golfeurs', 'city': 'Bussy-Saint-Georges'})
+
+@pytest.mark.qwant_non_regression
+def test_14_Ruelle_du_Moulin_Brûlé_77320():
```

Comment:
I'm not a big fan of these `û` & `é`

Reply:
https://www.python.org/dev/peps/pep-3131/ :stuck_out_tongue_winking_eye: